### PR TITLE
Stream Utilities

### PR DESCRIPTION
**Pull request by WilliamsJack**
A couple of new features for the camera stream. Floating action buttons allow you to refresh and screenshot an individual stream.

### CODE REVIEW

File: js/sart.demo.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 // Populate SARTInterface with dummy data to demonstrate what a functional setup looks like
 function DemoMode() {
+	demo = true;
```
Comment (Conr86) [inline js/sart.demo.js]: How does JS work when it comes to variables without either a `let` or `var` statement? Honestly JS is such as mess I don't know if this is an issue.

File: js/sart.demo.js [hunk 2]
```diff
@@ -25,9 +26,10 @@ function DemoMode() {
 	distChart.update();
 
 	// Camera streams, load dummy images
-	$('.streamImage').each(function (index, value) {
-		value.src = 'images/demo_' + value.id + '.jpg';
-		value.className = "streamImage";
+	$('.streamImage').each(function (index) {
+		$(this).attr('src','https://picsum.photos/575/430?' + Math.random());
+		$(this).attr('refresh_src','https://picsum.photos/575/430?' + Math.random());
```
Comment (Conr86) [inline js/sart.demo.js]: These `Math.random()` calls be different? Is that intended?

File: js/sart.demo.js (hunk 1 above)
Comment (WilliamsJack) [reply js/sart.demo.js]: Didn't notice that. Appears to work anyway? I couldn't tell you _why_ it works... But it does.

Commit 4fbf4d9 shows what it should be.

File: js/sart.demo.js (hunk 2 above)
Comment (WilliamsJack) [reply js/sart.demo.js]: Indeed. The images get cached by the browser for the session - we have to add some randomness to make the browser reload it.

The refresh_src is used by the refreshy button. If it was the same as the original src, they wouldn't refresh.

The motion stream doesn't appear to need this - the online service must have different cache control headers.

**Comment by WilliamsJack** [comment]
What a lovely review feature!

**Comment by Conr86** [comment]
Sure is. I like it 😁

**Pull request merged**